src/components/user/UserCardComponent.tsx

```diff
@@ -27,16 +29,16 @@ function stringAvatar(name: string) {
 
 function UserCardComponent() {
   const user = auth?.currentUser;
-
   const theme = useTheme();
+  const navigate = useNavigate();
 
   return (
     <>
       <Box
         sx={{
           display: 'flex',
```

Comment:
Maybe something like this? I'm not sure, but it looks better and more space for a nickname.
![image](https://user-images.githubusercontent.com/49300253/171587274-999dbdb9-70c0-458c-bba3-738a50675f02.png)